docs/source/examples/setup_your_own_data.ipynb

```diff
@@ -0,0 +1,862 @@
+{
```

Comment:
<p>This is looking great! Glad to see that <code>intake-esm</code> is useful here :)</p><p> </p><p> </p><p><code>ds = dset_dict[list(dset_dict.keys())[0]].load()</code></p><p> </p><p>Can be replaced with:</p><p> </p><p><code>_, ds = dset_dict.popitem()</code></p><p> </p><p> </p><p>And is it not the case that you don't want to call <code>.load()</code> here if you provided chunks</p><p> </p><p> </p><p><code>cdf_kwargs = {'chunks': {'time': 12}, 'decode_times': False}</code></p><p> </p><p>??</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bradyrx/climpred/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
You are right. But here I had do do it. I did `.load()` because climpred doesnt allow chunks in some dims in compute_perfect_model.